src/components/ExportDataModal.js

```diff
@@ -0,0 +1,208 @@
+    onDismiss: PropTypes.func.isRequired,
+    selectedEntities: PropTypes.array.isRequired,
+    selectedDataType: PropTypes.string.isRequired,
+    runningSubmissionsCount: PropTypes.number.isRequired
+  }
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      hardConflicts: [],
+      softConflicts: [],
+      additionalDeletions: [],
+      selectedWorkspaceId: undefined,
+      error: undefined,
+      copying: false
+    }
+  }
+
+  getSelectedWorkspace() {
+    const { workspaces } = this.props
+    const { selectedWorkspaceId } = this.state
+    return _.find({ workspace: { workspaceId: selectedWorkspaceId } }, workspaces)
+  }
+
+  render() {
+    const { copied } = this.state
+    return copied ? this.renderPostCopy() : this.renderCopyForm()
+  }
+
+  renderCopyForm() {
+    const { onDismiss, selectedEntities, runningSubmissionsCount, workspace, workspaces } = this.props
+    const { copying, hardConflicts, softConflicts, error, selectedWorkspaceId, additionalDeletions } = this.state
+    const moreToDelete = !!additionalDeletions.length
+    const warningStyle = {
+      border: `1px solid ${colors.orange[1]}`, borderLeft: 'none', borderRight: 'none',
+      backgroundColor: colors.orange[4],
+      padding: '1rem 1.25rem', margin: '0 -1.25rem',
+      color: colors.orange[0], fontWeight: 'bold', fontSize: 12
+    }
+    const errorStyle = {
+      ...warningStyle,
+      border: `1px solid ${colors.red[1]}`,
+      backgroundColor: colors.red[4],
+      color: colors.red[0]
+    }
+
+    const errors = validate(
+      { selectedWorkspaceId },
+      { selectedWorkspaceId: { presence: true } },
+    )
+
+    return h(Modal, {
+      onDismiss,
+      title: 'Copy to Workspace',
+      okButton: buttonPrimary({
+        tooltip: (hardConflicts.length !==0) ? 'Override existing entities, are you sure?' : Utils.summarizeErrors(errors),
+        disabled: !!errors || copying,
+        onClick: () => this.copy()
+      }, ['Copy'])
+    }, [
+      runningSubmissionsCount > 0 && InfoTile({
+        infoStyle: warningStyle, iconName: 'warning-standard',
+        content: `WARNING: ${runningSubmissionsCount} workflows are currently running in this workspace. ` +
+          'Copying the following data could cause failures if a workflow is using this data.'
+      }),
+      requiredFormLabel('Destination'),
+      h(WorkspaceSelector, {
+        workspaces: _.filter(({ workspace: { workspaceId }, accessLevel }) => {
+          return workspace.workspaceId !== workspaceId && Utils.canWrite(accessLevel)
+        }, workspaces),
+        value: selectedWorkspaceId,
+        onChange: v => this.setState({ selectedWorkspaceId: v })
+      }),
+      (hardConflicts.length !== 0) && InfoTile({
+        infoStyle: errorStyle, iconName: 'error-standard',
+        content: 'The following entries already exist in the selected workspace. Please select CANCEL to go back or COPY to override the existing entities. '
```

Comment:
How about: "Some of the following data already exists in the selected workspace. Click CANCEL to go back or COPY to override the existing data."
Thoughts:
- I think we want to avoid using the word "entities" as it doesn't mean much to users. 
- If a user selects 4 pieces of data and 2 pieces are duplicates, they will see this message. Correct? In which case I'd rephrase to "Some of"
- IMO a user _selects_ data and _clicks_ COPY. It's a semantic note but I figured I'd put it in while I'm here :)